src/llmq/utils.cpp

```diff
@@ -135,11 +163,16 @@ std::vector<CDeterministicMNCPtr> GetAllQuorumMembers(Consensus::LLMQType llmqTy
 
 std::vector<CDeterministicMNCPtr> ComputeQuorumMembers(Consensus::LLMQType llmqType, const CBlockIndex* pQuorumBaseBlockIndex)
 {
-    auto allMns = deterministicMNManager->GetListForBlock(pQuorumBaseBlockIndex);
-    auto modifier = ::SerializeHash(std::make_pair(llmqType, pQuorumBaseBlockIndex->GetBlockHash()));
     bool HPMNOnly = (Params().GetConsensus().llmqTypePlatform == llmqType) && IsV19Active(pQuorumBaseBlockIndex);
     const auto& llmq_params_opt = GetLLMQParams(llmqType);
     assert(llmq_params_opt.has_value());
+    assert(!llmq_params_opt->useRotation);
```

Comment:
why this assert?

Reply:
@PastaPastaPasta That's because `ComputeQuorumMembers` is for non-rotated quorums.

Reply:
and this calls this function for non-rotation no? 

https://github.com/dashpay/dash/blob/83236e3d5a19c8bce8467257d33d596b02c76b0a/src/llmq/utils.cpp#L156

Reply:
@PastaPastaPasta what about now ?